onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1247,6 +1247,8 @@ TEST(ExecutionProviderTest, FunctionTest) {
   status = session_object_2.Load(model_file_name);
   ASSERT_TRUE(status.IsOK());
   status = session_object_2.Initialize();
+  if (!status.IsOK())
+    std::cout << "ErrorMessage: " << status.ErrorMessage() << std::endl;
   ASSERT_TRUE(status.IsOK());
   status = session_object_2.Run(run_options, feeds, output_names, &fetches);
```

Comment:
`#include "test/util/include/asserts.h"` and with that ASSERT_STATUS_OK will do all this in one line.

Reply:
applied.